src/frontend/kernel/dune

```diff
@@ -0,0 +1,18 @@
+(library
+ (name query_protocol_kernel)
+ (public_name merlin-lib.query_protocol_kernel)
+ (libraries yojson ppx_yojson_conv_lib)
+ ; If we used any PPXes normally (for example by adding a `(preprocess (pps ppx_jane))`
+ ; clause), this would cause ppxlib to complain about not recognizing `yojson` as a valid
+ ; deriver, as described in this issue:
+ ; https://github.com/ocaml-ppx/ppxlib/issues/342
+ ; But in the foreseeable future we will not use any PPXes normally, so this is a
+ ; non-issue.
+ (lint
+  (pps ppx_yojson_conv)))
+
+; [@@deriving_inline] gets expanded by the `@lint` dune alias. This makes `@lint` get run
+; when we run the `@runtest` alias, which aides development.
+(alias
+  (name runtest)
```

Comment:
It would be convenient for this to get run as part of the `default` alias. But there's two things that give me pause:
1. I'm not sure if that will make opam try to run the ppx during compilation.
2. Maybe this is bad practice since `ppx_yojson_conv` is a `with-test` dependency, and this would make that a bit of a lie.